dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7806,6 +7806,7 @@ public String toString() {
               + "components including clients, workers, proxies etc. Note that Alluxio 3 will "
               + "eventually only support DORA structure, so this property will only serve "
               + "for a short period for compatibility, and will be removed eventually.")
+          .setAlias("alluxio.dora.client.read.location.policy.enabled")
```

Comment:
Technically I should add it to https://github.com/Alluxio/alluxio/blob/main/dora/core/common/src/main/java/alluxio/conf/RemovedKey.java
However that `RemovedKey` seems not updated for a while. I figure maybe we should use a separate PR to consolidate all keys that are moved or changes in 3.0?